core/schemas/bifrost.go

```diff
@@ -159,14 +160,15 @@ type ImageContent struct {
 // BifrostResponse represents the complete result from any bifrost request.
 type BifrostResponse struct {
 	ID                string                     `json:"id,omitempty"`
-	Object            string                     `json:"object,omitempty"` // text.completion or chat.completion
+	Object            string                     `json:"object,omitempty"` // text.completion/chat.completion or text.completion.chunk/chat.completion.chunk (streaming)
```

Comment:
I did not get this - 

Ideally this stands for polymorphism - which can be achieved using embedding in Go.

So we can two structs

`BifrostResponse` and `BifrostStreamingResponse` - and common fields can be in `BifrostBaseResponse` ?

Reply:
yes yes can do that, I thought eveything is same, just one field name is different but yes having a parent base response will be much cleaner